Dockerfile

```diff
@@ -178,22 +178,22 @@ RUN --mount=type=cache,sharing=locked,target=${BAZEL_CACHE},uid=${UID},gid=${GID
     set -eux && \
     #
     # Build using the final Ray.jl destination
-    ln -s ${RAY_REPO} ${RAY_JLL_REPO}/deps/ray && \
+    ln -s ${RAY_REPO} ${RAY_JLL_REPO}/build/ray && \
     rm -rf ${RAY_JLL_PROJECT} && \
     ln -s ${RAY_JLL_REPO} ${RAY_JLL_PROJECT} && \
     #
     # Build ray_julia_jll
-    julia --project=${RAY_JLL_PROJECT} -e 'using Pkg; Pkg.build(verbose=true); Pkg.precompile(strict=true)' && \
+    julia --project=${RAY_JLL_PROJECT} -e 'using Pkg; Pkg.build(); Pkg.precompile(strict=true)' && \
```

Comment:
can this still be done w/ `Pkg.build` or do we need to actually `include` the build.jl file?

Reply:
The `Dockerfile` needs to include the `build.jl` now and should have included the `build_jll.jl` when that was added. Having #155 would help catch these things in the future. I'll be making another PR which fixes the Dockerfile

Reply:
Opened #160 for this. As the Dockerfile is currently broken on `main` I think I'll get this merged so we can keep things moving.